src/db/collections/ledger_update.rs

```diff
@@ -19,19 +20,21 @@ use crate::{
 };
 
 /// Contains all informations related to an output.
+#[allow(missing_docs)]
```

Comment:
remove now?

Reply:
No, because then I would have to document the fields and I don't want to.

Reply:
right